docs/README.ja.md

```diff
@@ -1 +1,102 @@
-TODO
\ No newline at end of file
+<p align="center"><img src="/src/icons/icon@128.png" alt="holo-schedule logo"></p>
+
+# holo-schedule
+
+![Test status](https://github.com/YunzheZJU/holo-schedule/workflows/Test/badge.svg)
+
+—— Keep in touch with Hololive.
+
+ホロライブの配信を表示するブラウザ拡張です。
+
+[Eng](/README.md) [中文](docs/README.zh_CN.md)
```

Comment:
Let's prefer the absolute path. This fixes the broken link to `README.zh_CN.md`.
```suggestion
[Eng](/README.md) [中文](/docs/README.zh_CN.md)
```